apps/crm/src/modules/providers/providers-list/providers-list.tsx

```diff
@@ -0,0 +1,60 @@
+import { ProvidersTypes, ProviderType } from "../providers-types";
+import MailchimpLogo from "../../../assets/mailchimp.svg";
+import SegmentLogo from "../../../assets/semgent.png";
+import RudderstackLogo from "../../../assets/rudderstack.png";
+import { HTMLAttributes } from "react";
+import clsx from "clsx";
+import styles from "./providers-list.module.css";
+import { Text, Box, MarketplaceIcon } from "@saleor/macaw-ui/next";
+import { TextLink } from "../../ui/text-link/text-link";
+
+type Props = {
+  onProviderClick(provider: ProviderType): void;
+  activeProvider?: ProviderType;
+} & HTMLAttributes<HTMLUListElement>;
+
+export const ProvidersList = ({ className, onProviderClick, activeProvider, ...props }: Props) => {
+  return (
+    <ul className={clsx(className, styles.list)}>
+      <li
+        className={clsx(styles.item, {
+          [styles.activeItem]: activeProvider === "Mailchimp",
+        })}
+        onClick={() => {
+          onProviderClick("Mailchimp");
+        }}
+      >
+        <img alt="Mailchimp logo" className={styles.logo} src={MailchimpLogo.src} />
```

Comment:
**suggestion**: we have [`Avatar.User`](https://macaw-ui-next.vercel.app/?path=/story/components-avatar--user-with-image) or [`Avatar.Store`](https://macaw-ui-next.vercel.app/?path=/story/components-avatar--store-with-image) components that could be used there

Reply:
I need a white bg there, opened an issue https://github.com/saleor/macaw-ui/issues/339